### ISSUE TITLE
Close board and connection to Arduino

### ISSUE THREAD

**Issue by troywweber7** [firmata/firmata.js#133]
I've been browsing the firmata.js docs and code and I cannot seem to find a function that disconnects from the Arduino. For example

```javascript
let board = new firmata.Board('/dev/arduino' function(err){};);
```

starts up the connection to the Arduino board, but how do you close that connection? Do you simply `delete board`? Or is there some `firmata.disconnect(board)` function I'm missing?

**Comment by troywweber7** [comment]
The closest I found was `board.reset()` but this does not appear to disconnect from the serial device altogether.

**Comment by ericogr** [comment]
Maybe add an exit function:

```
Board.prototype.exit = function(callback) {
  this.transport.close(callback);
};
```

?

**Comment by ericogr** [comment]
#135

**Comment by troywweber7** [comment]
Sorry I never responded on here. That worked for me.

**Comment by troywweber7** [comment]
marking as closed since #135 is open.
